server/utils/citations.js

```diff
@@ -33,6 +32,8 @@ const getCollectionLevelData = (primaryCollectionPub) => {
 };
 
 export const generateCitationHTML = (pubData, communityData) => {
+	// eslint-disable-next-line global-require
```

Comment:
This is my attempt to prevent `citation-js` from loading during tests, because it creates a subprocess that doesn't terminate (why???) and Jest complains that it can't exit.